types/react-navigation/index.d.ts

```diff
@@ -10,8 +10,9 @@
 //                 charlesfamu <https://github.com/charlesfamu>
 //                 Tim Wang <https://github.com/timwangdev>
 //                 Qibang Sun <https://github.com/bang88>
+//                 Sergei Butko: <https://github.com/svbutko>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
+// TypeScript Version: 2.6
```

Comment:
Incrementing this version means you're using features only available in TypeScript version >=2.6, but I don't think this is true for your change. Unless there's something I'm not seeing, please revert this line. Thanks!

Reply:
Sorry, my mistake, reverted this line.